### PR TITLE
jour-479 allow display of preview templates

### PR DESCRIPTION
**Pull request by jsaleigh**
@yntema when you have a chance

### CODE REVIEW

File: src/6_branch.js [hunk 1]
```diff
@@ -272,6 +272,15 @@ Branch.prototype['init'] = wrap(
 
 		options = (options && typeof options === 'function') ? { } : options;
 
+		var branchViewId = null;
+
+		if (options) {
+			branchViewId = options.branch_view_id || null;
+		}
+		if (!branchViewId) {
+			branchViewId = utils.getParameterByName('_branch_view_id') || null;
```
Comment (yntema) [inline src/6_branch.js]: Should the underscore be removed from '_branch_view_id' ?

Comment (jsaleigh) [inline src/6_branch.js]: I did that intentionally, to reduce the chance of collision with partner QSPs.  It might not be needed though.

Comment (zeeshabnc) [inline src/6_branch.js]: Lets leave the underscore, gives the implication that its internal use

**Comment by yntema** [comment]
👍

File: src/6_branch.js (hunk 1 above)
Comment (yntema) [inline src/6_branch.js]: Cool, just wanted to make sure it was intentional

**Pull request merged**